.changeset/popular-oranges-end.md

```diff
@@ -0,0 +1,6 @@
+---
+"@latticexyz/store": patch
+"@latticexyz/cli": patch
+---
+
+Fixed the behaviour of static arrays, so that they always return zero for uninitialised values. Previously they would revert with `Store_IndexOutOfBounds` if the index had not been set yet.
```

Comment:
Should we make a note about this matching native solidity behavior?

Reply:
I like that! lemme think